Podfile

```diff
@@ -1,4 +1,4 @@
-platform :ios, '10.0'
+platform :ios, '11.0'
```

Comment:
Is this needed to get the pods to work? Otherwise let's not change it in the PR.

Reply:
i changed because it doesnt work without this update.